extension.ts

```diff
@@ -328,6 +328,22 @@ const createRecentItem = (kind: RecentItemKind) => (
   };
 };
 
+/**
+ * Convert any workspace item objects to URIs.
+ *
+ * @param item The workspace item from storage.json
+ * @returns The URI for the workspace
+ */
+const workspaceItemToUri = (
+  item: string | { configURIPath: string }
+): string => {
+  if (typeof item === "object" && "configURIPath" in item) {
```

Comment:
Invert the condition (i.e. `typeof item !== 'string'`) to simplify it; typescript then automatically figures out what properties the object has, and there's no need for the `in` test.

Reply:
This was to be more cautious. I only know that these two variations exist, but there might be more that I haven't seen, so I prefer making sure that the key exists before reading it.

Reply:
If you're not sure what you get the argument type should be `unknown` shouldn't it?